plugins/modules/remote_management/redfish/redfish_command.py

```diff
@@ -207,6 +207,15 @@
         description:
           - The transfer method to use with the image
         type: str
+  strip_etag_quotes:
+    description:
+      - Removes surrounding quotes of etag used in C(If-Match) header
+        of C(PATCH) requests.
+      - Only use this option to resolve bad vendor implementation where
+        C(If-Match) only matches the unquoted etag string.
+    type: bool
+    default: false
+    version_added: 3.6.0
```

Comment:
Since 3.6.0 has already been released, this needs to be updated:
```suggestion
    version_added: 3.7.0
```